File: SoloLearn Codes/Duct Tape/DuctTape.java
import java.util.Scanner;

public class DuctTape
{
    public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
        int height = sc.nextInt();
        int width = sc.nextInt();
        System.out.println((int) Math.ceil(((width * height) * 2) / 10.0));
	}
}

File: SoloLearn Codes/Duct Tape/DuctTape.cs
using System;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int height = Convert.ToInt32(Console.ReadLine());
            int width = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine((int) Math.Ceiling((double)(height * width * 2) / 10));
        }
    }
}

